src/ralph/ssl_certificates/management/commands/update_service_env.py

```diff
@@ -0,0 +1,58 @@
+import sys
+
+from django.core.management.base import BaseCommand
+
+from ralph.assets.models.assets import ServiceEnvironment
+from ralph.dns.dnsaas import dnsaas_client
+from ralph.ssl_certificates.models import SSLCertificate
+
+
+class Command(BaseCommand):
+    help = 'Checks the compliance of services in SSL Certificates'
+
+    def update_from_record(self, results):
+        for value in results:
+            domain = value['content']
```

Comment:
isn't `content` IP address (at least for A records)? shouldn't you look at `name` field?

Reply:
You have absolutely right. It's depend of record type. I'll change it.